src/DataCite.jl

```diff
@@ -0,0 +1,52 @@
+struct DataCite <: DataRepo
+end
+
+function description(repo::DataCite, mainpage)
+    attributes = mainpage["attributes"]
+    desc = attributes["description"]
+    authors = join.([[names[2] for names in value] for value in attributes["author"]], " ")
+    author = format_authors(authors)
+    license = attributes["license"]
+    date = attributes["published"]
+    paper = format_papers(authors, date, attributes["title"] * " [Data set]. " * attributes["container-title"] * ".", mainpage["id"])
```

Comment:
Anytime we have a DOI and want a citation we should do it via 
method discussed in https://github.com/oxinabox/DataDepsGenerators.jl/issues/22#issuecomment-392708560

This is fine for now,
but maybe after this PR make another one that does through and replaces all paper formatting with something based on that kind of idea?

Reply:
That's the reason I introduced the `format_papers()` method. I subsequently can add whatever format is required.